users/migrations/0001_initial.py

```diff
@@ -45,6 +45,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='user',
             name='avatar',
-            field=models.ImageField(upload_to='images/'),
+            field=models.ImageField(blank=True, upload_to='images/'),
```

Comment:
If the migration is already in the master branch do not change it, make new
Find an answer to why this case can break all system